cleaners/vlc.xml

```diff
@@ -19,20 +17,68 @@
+    @tested ok v???, Windows 7
+    @testeddate 2019-03-10 (... with an old version of the cleaner!)
+    @testedby https://github.com/Tobias-B-Besemer
+    @note 
+
 -->
 <cleaner id="vlc">
   <label>VLC media player</label>
   <description>Media player</description>
+  <!-- Don't know if the check of the task is really necessary... Tobias. -->
   <running type="exe" os="windows">vlc.exe</running>
   <running type="exe" os="linux">vlc</running>
+  <!--
+  BSD tasks not yet figured out!
+  <running type="exe" os="FreeBSD">vlc</running>
+  <running type="exe" os="NetBSD">vlc</running>
+  <running type="exe" os="OpenBSD">vlc</running>
+  Guess it would be:
+  <running type="exe" os="unix">vlc</running>
+  -->
+  <var name="Profile">
+    <value os="windows">%AppData%\vlc</value>
+    <value os="linux">$XDG_CONFIG_HOME/vlc</value>
+    <!--
+    BSD $$Profile$$ not yet figured out!
+    <value os="FreeBSD">$XDG_CONFIG_HOME/vlc</value>
+    <value os="NetBSD">$XDG_CONFIG_HOME/vlc</value>
+    <value os="OpenBSD">$XDG_CONFIG_HOME/vlc</value>
+    Guess it would be:
+    <value os="unix">$XDG_CONFIG_HOME/vlc</value>
+    -->
+  </var>
+  <var name="Config-File">
+    <value os="windows">vlc-qt-interface.ini</value>
+    <value os="linux">vlc-qt-interface.conf</value>
+    <!--
+    BSD $$Config-File$$ not yet figured out!
+    <value os="FreeBSD">vlc-qt-interface.conf</value>
+    <value os="NetBSD">vlc-qt-interface.conf</value>
+    <value os="OpenBSD">vlc-qt-interface.conf</value>
+    Guess it would be:
+    <value os="unix">vlc-qt-interface.conf</value>
+    -->
+  </var>
   <option id="mru">
     <label>Most recently used</label>
     <description>Delete the most recently used list</description>
-    <action search="file" path="$APPDATA\vlc\vlc-qt-interface.ini" command="ini" section="General" parameter="filedialog-path"/>
-    <action search="file" path="$APPDATA\vlc\vlc-qt-interface.ini" command="ini" section="RecentsMRL"/>
-    <action search="file" path="$XDG_CONFIG_HOME/vlc/vlc-qt-interface.conf" command="ini" section="General" parameter="filedialog-path"/>
-    <action search="file" path="$XDG_CONFIG_HOME/vlc/vlc-qt-interface.conf" command="ini" section="RecentsMRL"/>
-    <action search="file" path="$XDG_CONFIG_HOME/vlc/vlc-qt-interface.conf" command="ini" section="OpenDialog" parameter="netMRL"/>
-    <action command="delete" search="walk.files" path="$APPDATA\vlc\art\artistalbum\*"/>
+    <action search="file" path="$$Profile$$\$$Config-File$$" command="ini" section="General" parameter="filedialog-path"/>
+    <action search="file" path="$$Profile$$\$$Config-File$$" command="ini" section="RecentsMRL"/>
+    <action search="file" path="$$Profile$$\$$Config-File$$" command="ini" section="OpenDialog" parameter="netMRL"/>
+    <action command="delete" search="walk.files" path="$$Profile$$\art\artistalbum\*"/>
```

Comment:
Did you observe the artistalbum files here on Linux? They are not here on my Linux system. They are in  `~/.cache/vlc/art/artistalbum/`

I have VLC 3.0.4.

I am guessing for Linux you will not use $$profile$$. You can just type it directly without a variable.

Reply:
I will change that.

Reply:
I'm using now XDG_CACHE_HOME, is this right?

Reply:
Yes please

Reply:
On VLC 3.0.7.1 Vetinari the art is under `~./cache/vlc/art/artistalbum/`